gilda/term.py

```diff
@@ -9,7 +9,7 @@
     "filter_out_duplicates",
 ]
 
-logger = logging.getLogger("gilda.term")
+logger = logging.getLogger(__name__)
```

Comment:
I typically do this but noticed it wasn't the case in the other module, maybe we can normalize that one now too